### PR TITLE
Get rid of request custom attributes

### PR DESCRIPTION
**Pull request by leplatrem**
Tests won't pass until https://github.com/mozilla-services/kinto/pull/32 is merged.

### CODE REVIEW

File: cliquet/tests/resource/test_filter.py [hunk 1]
```diff
@@ -27,20 +27,20 @@ def test_list_can_be_filtered_on_deleted_with_since(self):
...
-        self.db.delete(self.resource, 'bob', r['id'])
+        r = self.storage.create(self.resource, 'bob', {})
+        self.storage.delete(self.resource, 'bob', r['id'])
         self.resource.request.GET = {'deleted': 'true'}
         result = self.resource.collection_get()
         self.assertEqual(len(result['items']), 0)
 
     def test_filter_works_with_empty_list(self):
-        self.resource.db_kwargs['user_id'] = 'alice'
+        self.resource.storage_kwargs['user_id'] = 'alice'
```
Comment (ametaireau) [inline cliquet/tests/resource/test_filter.py]: You [named it `storage_kw`](https://github.com/mozilla-services/cliquet/pull/245/files#diff-5443aeaa6b1f4c0c732f73d91e160625R80).

**Comment by ametaireau** [comment]
(updated)

**Comment by Natim** [comment]
Rebased.

**Comment by leplatrem** [comment]
I think a glitch was introduced during rebase. I'll fix it.

**Comment by Natim** [comment]
Thanks :)

**Pull request merged**